### PR TITLE
upd sql from unspent

### PR DESCRIPTION
**Pull request by vikingphoenix**
- removed dup endpoint
- updated sql query to match upspent service work

### CODE REVIEW

File: app/api/v1/routes/blockchain.py [hunk 1]
```diff
@@ -95,28 +95,40 @@ async def getInfo():
...
-    ''')
-    stats = engDanaides.execute(sql, {'token_id': tokenId}).fetchall()
-
-    res = {}
-    for stat in stats:        
-        res[stat['notes']] = stat['amount']
-
-    return res
+    try:
+        engDanaides = create_engine(CFG.csDanaides)
```
Comment (noob77777) [inline app/api/v1/routes/blockchain.py:101]: Suggestion: Have this cached even if the endpoint only takes a couple of seconds to reduce load on db and server
```python
cached = cache.get(f"get_api_blockchain_tokenomics_{tokenId}")
        if cached:
            return cached
```

and 

```python
cache.set(f"get_api_blockchain_tokenomics_{tokenId}", stats)
```

**Review by noob77777 — COMMENTED** [review]
Can update the frontend once the delta is synced.

File: app/api/v1/routes/blockchain.py (hunk 1 above)
Comment (noob77777) [reply app/api/v1/routes/blockchain.py:101]: https://github.com/ergo-pad/ergopad-api/pull/348

**Comment by vikingphoenix** [comment]
as long as it's not slower...ideally, the engine is pooled and kept open

**Pull request merged**